static/js/appjs/user/profile.js

```diff
@@ -603,6 +603,31 @@
             }
         });
 
+        $('#friend-list-modal').modal();
+
+        $('#friend-list-button').click(function() {
+            var $this = $(this)
+            if (isLoggedIn) {
+                $('#friend-list-modal').modal('open');
```

Comment:
make the ajax call first and then open the modal on success, will avoid the immediate expansion of the modal due to the data.